### PR TITLE
Add account_signers table

### PR DESCRIPTION
**Pull request by erika-sdf**

### CODE REVIEW

File: dags/bucket_list_dag.py [hunk 1]
```diff
@@ -45,43 +46,48 @@
...
 The write batch stats task will take a snapshot of the DAG run_id, execution date,
 start and end ledgers so that reconciliation and data validation are easier. The
 record is written to an internal dataset for data eng use only.
 '''
-write_acc_stats = build_batch_stats(dag, 'accounts')
-write_bal_stats = build_batch_stats(dag, 'claimable_balances')
-write_off_stats = build_batch_stats(dag, 'offers')
-write_pool_stats = build_batch_stats(dag, 'liquidity_pools')
-write_trust_stats = build_batch_stats(dag, 'trust_lines')
+write_acc_stats = build_batch_stats(dag, table_names['accounts'])
```
Comment (sydneynotthecity) [inline dags/bucket_list_dag.py:57]: Nice, I like this change. Gives us more flexibility if we change the table names in the future.

File: dags/fail_dag.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+'''
+This DAG export claimable balances from start to end date. It should be removed once it's caught up.
```
Comment (sydneynotthecity) [inline dags/fail_dag.py:2]: Nit: Can we change the description if this DAG is supposed to test failures?

File: dags/fail_dag.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    default_args=get_claimable_balances_dag_args(),
+    start_date=datetime.datetime(2021, 10, 1),
+    end_date=datetime.datetime(2022, 1, 13),
+    description='just fails every 5 min',
+    schedule_interval='0*/5 * * * *',
+    user_defined_filters={'fromjson': lambda s: json.loads(s)},
+)
+date_task = build_time_task(dag)
+
+fail_task = build_fail_task()
```
Comment (sydneynotthecity) [inline dags/fail_dag.py:34]: Is the `build_fail_task()` code part of the PR? Or is the intent for that to remain in a local repo?

File: dags/history_archive_without_captive_core_dag.py [hunk 4]
```diff
@@ -46,7 +46,7 @@
 
 '''
 The export tasks call export commands on the Stellar ETL using the ledger range from the time task.
-The results of the comand are stored in a file. There is one task for each of the data types that 
+The results of the command are stored in a file. There is one task for each of the data types that
```
Comment (sydneynotthecity) [inline dags/history_archive_without_captive_core_dag.py:49]: Should the table names in this DAG also be updated to use the `table_ids[<table_name>]` convention?

**Review by sydneynotthecity — COMMENTED** [review]
Couple of questions, but overall looks good

File: dags/fail_dag.py (hunk 2 above)
Comment (erika-sdf) [reply dags/fail_dag.py:2]: I meant to leave this dag out; removing from this PR

File: dags/fail_dag.py (hunk 3 above)
Comment (erika-sdf) [reply dags/fail_dag.py:34]: My mistake; this dag should have been left out

File: dags/bucket_list_dag.py [hunk 5]
```diff
@@ -45,43 +46,48 @@
...
 The apply tasks receive the location of the file in Google Cloud storage through Airflow's XCOM system.
 Then, the task merges the entries in the file with the entries in the corresponding table in BigQuery. 
 Entries are updated, deleted, or inserted as needed.
 '''
-send_acc_to_bq_task = build_gcs_to_bq_task(dag, export_acc_task.task_id, 'accounts', '', partition=False)
-send_bal_to_bq_task = build_gcs_to_bq_task(dag, export_bal_task.task_id, 'claimable_balances', '', partition=False)
-send_off_to_bq_task = build_gcs_to_bq_task(dag, export_off_task.task_id, 'offers', '', partition=False)
-send_pool_to_bq_task = build_gcs_to_bq_task(dag, export_pool_task.task_id, 'liquidity_pools', '', partition=False)
-send_trust_to_bq_task = build_gcs_to_bq_task(dag, export_trust_task.task_id, 'trustlines', '',  partition=False)
+send_acc_to_bq_task = build_gcs_to_bq_task(dag, export_acc_task.task_id, table_names['accounts'], '', partition=False)
```
[the remark this answers is not in the document]
Comment (erika-sdf) [reply dags/bucket_list_dag.py:80]: Sure. I'll start an issue for it

File: dags/history_archive_without_captive_core_dag.py (hunk 4 above)
Comment (erika-sdf) [reply dags/history_archive_without_captive_core_dag.py:50]: Good call. I've added them to the two `history_archive` dags

**Pull request merged**